debian/copyright

```diff
@@ -0,0 +1,18 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: rdiff-backup
+Upstream-Contact: rdiff-backup-users@nongnu.org
+Source: https://github.com/rdiff-backup/rdiff-backup
+
+Files: *
+Copyright: 2001-2008 Ben Escoto <bescoto@stanford.edu>
+           2019-2020 Eric Lavarde <eric@lavar.de>
```

Comment:
can you please use my rdiffbackup address? ewl+rdiffbackup-at-lavar.de

Reply:
ok